services/server/src/routes/fawry.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from "express";
+import fawry from "@/handlers/fawry";
+
+const router = Router();
+
+router.post("/pay/card-token", fawry.payWithCard);
+router.post("/pay/ref-num", fawry.payWithRefNum);
+router.post("/pay/e-wallet", fawry.payWithEWallet);
+router.post("/pay/bank-installments", fawry.payWithBankInstallments);
+
+router.post("/refund", fawry.refund);
+
+router.get("/card-token/list", fawry.listCardTokens);
+router.delete("/card-token/delete", fawry.deleteCardToken);
```

Comment:
We need an endpoint to cancel unpaid txs ([ref](https://developer.fawrystaging.com/docs/server-apis/cancel-payment-api))